Forte.EpiServer.AzureSearch/Events/SearchEventHandler.cs

```diff
@@ -12,92 +13,125 @@ namespace Forte.EpiServer.AzureSearch.Events
     public class SearchEventHandler<T> where T : ContentDocument
     {
         private readonly IAzureSearchService _azureSearchService;
-        private readonly IContentDocumentBuilder<T> _contentDocumentBuilder;
-        private readonly IContentLoader _contentLoader;
+        private readonly IContentRepository _contentRepository;
```

Comment:
I think we can keep IContentLoader - we only need IContentRepository when doing write operations on content

Reply:
True, it's a leftover from before splitting the searchEventHandler class to page and block ones.